### PR TITLE
feat(region,host): sriov nic support

### PR DESCRIPTION
**Pull request by wanyaoqi**
Support sriov nic passthrough to kvm guest.

Signed-off-by: wanyaoqi <d3lx.yq@gmail.com>

**What this PR does / why we need it**:

**Does this PR need to be backport to the previous release branch?**:
NONE

### CODE REVIEW

File: pkg/hostman/isolated_device/isolated_device.go [hunk 1]
```diff
@@ -327,6 +360,10 @@ func GetApiResourceData(dev IDevice) *jsonutils.JSONDict {
 	if len(dev.GetGuestId()) != 0 {
 		data["guest_id"] = dev.GetGuestId()
 	}
+	if len(dev.GetWireId()) != 0 {
+		log.Errorf("wire_id %s", dev.GetWireId())
```
Comment (swordqiu) [inline pkg/hostman/isolated_device/isolated_device.go:364]: log.Infof?

File: pkg/hostman/isolated_device/nic.go [hunk 2]
```diff
@@ -0,0 +1,194 @@
+package isolated_device
```
Comment (swordqiu) [inline pkg/hostman/isolated_device/nic.go:1]: License

File: pkg/hostman/guestman/qemu/generate.go [hunk 3]
```diff
@@ -373,6 +373,10 @@ func generateNicOptions(drvOpt QemuOptions, input *GenerateStartOptionsInput) ([
 	nics := input.GuestDesc.Nics
 
 	for idx := range nics {
+		if nics[idx].Driver == "vfio-pci" {
```
[the remark this answers is not in the document]
Comment (wanyaoqi) [reply pkg/hostman/guestman/qemu/generate.go]: fixed

File: pkg/hostman/isolated_device/nic.go (hunk 2 above)
Comment (wanyaoqi) [reply pkg/hostman/isolated_device/nic.go:15]: fixed

File: pkg/hostman/isolated_device/isolated_device.go (hunk 1 above)
Comment (wanyaoqi) [reply pkg/hostman/isolated_device/isolated_device.go]: 调试信息，漏了删，已移除

**Pull request merged**